### PR TITLE
added #89 populate project tree from project store

### PR DESCRIPTION
**Pull request by rohitgupta274509**
please review and approve it.

### CODE REVIEW

File: src/components/Dialog/Newconnections.vue [hunk 1]
```diff
@@ -74,92 +77,91 @@
...
-  height: 100vh;
-  width: 40vw;
-  .p-dropdown {
+<style lang="scss">
+  input {
     width: 80%;
   }
-  .p-dialog-content {
-    height: 100%;
+  .field {
```
Comment (ankitvats1) [inline src/components/Dialog/Newconnections.vue:144]: Do not modify Prime Vue Class Directly.

File: src/components/Dialog/Projects.vue [hunk 2]
```diff
@@ -59,83 +59,123 @@
...
-.field {
-  display: grid;
-}
-.projects-dialog {
-  height: 100vh;
-  width: 40vw;
-  .p-dropdown{
+<style lang="scss">
+  .dropdown1 {
+    width: 100% !important;
```
Comment (ankitvats1) [inline src/components/Dialog/Projects.vue:155]: Do not use !important, unless it is the only option left.

File: src/components/Dialog/Projects.vue [hunk 3]
```diff
@@ -59,83 +59,123 @@
...
+  }
+  input {
     width: 80%;
   }
-  .p-dialog-content {
-    height: 100%;
+  .field {
```
Comment (ankitvats1) [inline src/components/Dialog/Projects.vue:160]: Remove this from all files. (Refer 1st comment)

**Review by ankitvats1 — CHANGES_REQUESTED** [review]

File: src/components/Dialog/Newconnections.vue (hunk 1 above)
Comment (rohitgupta274509) [reply src/components/Dialog/Newconnections.vue:144]: this issue have been resolved in ticket #149

File: src/components/Dialog/Projects.vue (hunk 2 above)
Comment (rohitgupta274509) [reply src/components/Dialog/Projects.vue:155]: this issue have been resolved in ticket #149

File: src/components/Dialog/Projects.vue (hunk 3 above)
Comment (rohitgupta274509) [reply src/components/Dialog/Projects.vue:160]: this issue have been resolved in ticket #149

**Review by ankitvats1 — APPROVED** [review]

**Review by Shivam-apna — APPROVED** [review]

**Review by Rahul139-stack — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -45,6 +45,8 @@
     "@achrinza/node-ipc": "^10.1.6",
     "@mdi/font": "5.9.55",
     "@rauschma/stringio": "^1.4.0",
+    "@vuelidate/core": "^2.0.0-alpha.41",
```
Comment (ankitvats1) [inline package.json:48]: This code has already been merged. Why is it showing again?

**Review by ankitvats1 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**